src/server/status/metrics.js

```diff
@@ -1,40 +1,49 @@
-import _ from 'lodash';
+import { get, set, isObject } from 'lodash';
 import Samples from './samples';
 import { keysToSnakeCaseShallow } from '../../utils/case_conversion';
+import { getAllStats } from './cgroup';
 
 export function collectMetrics(kbnServer, server, config) {
   server.plugins['even-better'].monitor.on('ops', function (event) {
-    kbnServer.metrics = getMetrics({ event, config });
+    getMetrics({ event, config }).then(data => { kbnServer.metrics = data; });
   });
 }
 
-export function getMetrics({ event, config }) {
+export async function getMetrics({ event, config }) {
   const port = config.get('server.port');
   const timestamp = new Date().toISOString();
-  return {
+  const cgroup = await getAllStats();
```

Comment:
Now preventing lookups when we determine there is no cgroups information. https://github.com/elastic/kibana/pull/10402/commits/655f54bc3b113f08c88c5ee24d3e3c4a00125dcc